setup.py

```diff
@@ -89,11 +89,11 @@ def run(self):
 
 requirements = [
     "galaxy-importer==0.4.2",
-    "pulpcore>=3.15.0,<3.16.0",
-    "pulp-ansible>=0.10.1,<0.11.0",
+    "pulpcore>=3.17.1,<3.18.0",
+    "pulp-ansible>=0.11.0,<0.12.0",
```

Comment:
should this minimum `pulp-ansible>=0.11.0` and template_config.yml value `branch: 0.11.1` match?

Reply:
good catch!